lib/datadog/appsec/assets/waf_rules/recommended.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
:+1: I've checked the sha256 of the new files match the ones in https://github.com/datadog/appsec-event-rules/tree/1.5.0:

0be5309e8c0ea542b3b2aee4b367144573c1d0921a9e78e021a9ba93b3418db2  recommended.json
d302af9ee296b28742e38015077a76e0786fa82537683f4d6b18ad8993d1ab4d  strict.json